src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireControllerTest.java

```diff
@@ -36,279 +37,349 @@
+        .andExpect(
+            MockMvcResultMatchers.jsonPath("$[1].nom", CoreMatchers.is(proprietaireDTO2.getNom())))
+        .andExpect(MockMvcResultMatchers.jsonPath("$[1].prenom",
+            CoreMatchers.is(proprietaireDTO2.getPrenom())))
+        .andExpect(MockMvcResultMatchers.jsonPath("$[1].email",
+            CoreMatchers.is(proprietaireDTO2.getEmail())))
+        .andExpect(MockMvcResultMatchers.jsonPath("$[1].numTel",
+            CoreMatchers.is(proprietaireDTO2.getNumTel())))
+        .andExpect(MockMvcResultMatchers.jsonPath("$[1].logements",
+            CoreMatchers.is(proprietaireDTO2.getLogements())));
+  }
+
+  @Test
+  public void ProprietaireController_GetProprietaire_WithIdInPath_ReturnOk() throws Exception {
+    given(proprietaireService.getProprietaireById(1L, false)).willAnswer(
+        invocationOnMock -> proprietaireDTO);
+
+    when(authService.verifyToken(any(), anyBoolean())).thenReturn(
+        new AbstractMap.SimpleEntry<>(proprietaireDTO.getId(), true)); //skip token verif
+
+    ResultActions response = mockMvc.perform(get("/proprietaires/1")
+        .header("Authorization",
+            "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJqb2huZG9lQGdtYWlsLmNvbTs2MjZjMjlhNS0zZGU0LTQ5Y2YtODI4ZS1hNDkxNWQzMzM5N2EiLCJpYXQiOjE3MTQxMTM1NzcsImV4cCI6MTcxNDExNzE3N30.kbU1pVkUHkRXktX44JBFN_xzDv-ZvSmtPjnjORO0vHPiHd3f2MGfDF15VTFO5icIrU_bV9cTqZ70RDlKdlp0-g")
+        .contentType(MediaType.APPLICATION_JSON));
+
+    response.andExpect(MockMvcResultMatchers.status().isOk())
+        .andExpect(
+            MockMvcResultMatchers.jsonPath("$.nom", CoreMatchers.is(proprietaireDTO.getNom())))
+        .andExpect(MockMvcResultMatchers.jsonPath("$.prenom",
+            CoreMatchers.is(proprietaireDTO.getPrenom())))
+        .andExpect(
+            MockMvcResultMatchers.jsonPath("$.email", CoreMatchers.is(proprietaireDTO.getEmail())))
+        .andExpect(MockMvcResultMatchers.jsonPath("$.numTel",
+            CoreMatchers.is(proprietaireDTO.getNumTel())))
+        .andExpect(MockMvcResultMatchers.jsonPath("$.logements",
+            CoreMatchers.is(proprietaireDTO.getLogements())));
+  }
+
+  @Test
+  public void ProprietaireController_UpdateProprietaire_ReturnOk() throws Exception {
+    proprietaireDTO.setPrenom(proprietaireUpdateDTO.getPrenom());
+    given(proprietaireService.updateProprietaire(1L, proprietaireUpdateDTO)).willAnswer(
+        invocationOnMock -> proprietaireDTO);
+
+    when(authService.verifyToken(any(), anyBoolean())).thenReturn(
+        new AbstractMap.SimpleEntry<>(proprietaireDTO.getId(), true)); //skip token verif
+
+    ResultActions response = mockMvc.perform(put("/proprietaires/1")
+        .header("Authorization",
+            "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJqb2huZG9lQGdtYWlsLmNvbTs2MjZjMjlhNS0zZGU0LTQ5Y2YtODI4ZS1hNDkxNWQzMzM5N2EiLCJpYXQiOjE3MTQxMTM1NzcsImV4cCI6MTcxNDExNzE3N30.kbU1pVkUHkRXktX44JBFN_xzDv-ZvSmtPjnjORO0vHPiHd3f2MGfDF15VTFO5icIrU_bV9cTqZ70RDlKdlp0-g")
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 261).